Remove `admin-only` label from topology related procedures (#1748)
neo4j/docs-operations: modules/ROOT/pages/reference/procedures.adoc

| xref:reference/procedures.adoc#procedure_dbms_cluster_setAutomaticallyEnableFreeServers[`dbms.cluster.setAutomaticallyEnableFreeServers()`]
| label:no[]
| label:yes[]

> | 

Author: ```suggestion
|  label:admin-only[Admin only before 5.23]
```

> |  label:admin-only[Admin only before 5.23]

Author: ```suggestion
| Before Neo4j 5.23, the procedure can be run only with the `Admin` privileges.
```

| xref:reference/procedures.adoc#procedure_dbms_cluster_cordonServer[`dbms.cluster.cordonServer()`]
| label:no[]
| label:yes[]

> | 

Author: ```suggestion
| Before Neo4j 5.23, the procedure can be run only with the `Admin` privileges.
```

| xref:reference/procedures.adoc#procedure_dbms_cluster_uncordonServer[`dbms.cluster.uncordonServer()`]
| label:no[]
| label:yes[]

> | label:deprecated[Deprecated in 5.23]. +

Author: ```suggestion
| label:deprecated[Deprecated in 5.23]. +
Before Neo4j 5.23, the procedure can be run only with the `Admin` privileges. +
```

m|WRITE
|===

[NOTE]

> ====
> Before Neo4j 5.23, the procedure `dbms.cluster.setAutomaticallyEnableFreeServers()` can be run by users with the `Admin` privileges only.
> ====

Reviewer: Suggestion for more detail:
```suggestion
====
Before Neo4j 5.23, the procedure `dbms.cluster.setAutomaticallyEnableFreeServers()` can be run by users with the `Admin` privileges only.
Since Neo4j 5.23, it can be run with the `SERVER MANAGEMENT` privilege.
It will still run with the `Admin` privilege, but that should be considered deprecated.
====
```

Reviewer: (This is true for all three procedures)

m|WRITE
|===

[NOTE]
====

> Before Neo4j 5.23, the procedure `dbms_cluster_cordonServer()` can be run only by users with `Admin` privileges.

Reviewer: ```suggestion
Before Neo4j 5.23, the procedure `dbms.cluster.cordonServer()` can be run only by users with `Admin` privileges.
```
